### PR TITLE
fix: properly stop all previous tracks during sfu migration

### PR DESCRIPTION
**Pull request by raviteja83**
### Details(context, link the issue, how was the bug fixed, what does the new feature do)

-
-

### CODE REVIEW

File: packages/hms-video-store/src/connection/HMSConnection.ts [hunk 1]
```diff
@@ -28,6 +28,8 @@ export default abstract class HMSConnection {
    *  - [HMSSubscribeConnection] clears this list as soon as we call [addIceCandidate]
    */
   readonly candidates = new Array<RTCIceCandidateInit>();
+  // @ts-ignore
```
Comment (KaustubhKumar05) [inline packages/hms-video-store/src/connection/HMSConnection.ts:31]: Why was this needed?

**Review by KaustubhKumar05 — APPROVED** [review]

File: packages/hms-video-store/src/connection/HMSConnection.ts (hunk 1 above)
Comment (raviteja83) [reply packages/hms-video-store/src/connection/HMSConnection.ts:31]: property is not read here hence needed

**Review by eswarclynn — APPROVED** [review]

**Review by KaustubhKumar05 — APPROVED** [review]
